include/rdm/defs.h

```diff
@@ -743,3 +744,45 @@
 #define E137_7_NR_ENDPOINT_NUMBER_INVALID                0x000C /* The specified endpoint is invalid. */
 #define E137_7_NR_INVALID_ENDPOINT_MODE                  0x000D /* The specified endpoint is in an invalid Endpoint Mode for the requested action. */
 #define E137_7_NR_UNKNOWN_UID                            0x000E /* The specified UID is not recognized. */
+
+
+/********************************************************************************************************************************/
+/********************************************************************************************************************************/
+/* ANSI E1.33 DEFINES */
+/********************************************************************************************************************************/
+/********************************************************************************************************************************/
+
+/******************************************************************************
+ * E1.33 Table A-15: RDM Parameter ID Defines
+ *****************************************************************************/
```

Comment:
This is just nitpicking, but we should make the block comments here match the style that they appear in the rest of the file. This file was originally obtained from an external source (https://www.rdmprotocol.org/rdm/developers/developer-resources/) so it's OK to maintain consistency with its style, against our usual style. Same goes for the indentation of the #defines below.